src/main/java/org/opensearch/flowframework/rest/RestGetWorkflowStepAction.java

```diff
@@ -91,7 +92,9 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
                     XContentBuilder exceptionBuilder = ex.toXContent(channel.newErrorBuilder(), ToXContent.EMPTY_PARAMS);
                     channel.sendResponse(new BytesRestResponse(ex.getRestStatus(), exceptionBuilder));
                 } catch (IOException e) {
-                    String errorMessage = "IOException: Failed to send back get workflow step exception";
+                    String errorMessage = ParameterizedMessageFactory.INSTANCE.newMessage(
+                        "IOException: Failed to send back get workflow step exception"
```

Comment:
see above; no need to replace a plain string but consider parameterizing "get workflow step"